test/integration/formatting_stack/strategies.clj

```diff
@@ -49,7 +61,7 @@
       (-> deletable-file .delete)
       (expect-sane-output! (sut/all-files :files []))
       (finally
-        (sh "git" "checkout" (str deletable-file))
+        (sh "git" "checkout" ".")
```

Comment:
Other tests already relied on `.`, since we run these commands in a copied (temporary) repo **and** assert `git pristine` before, i think this is fine.

Reply:
I don't know, it would be very frustrating if some programming error meant that one's WIP was erased.

Reply:
True, but I don't think it's likely. However; simply introducing another constant here should be doable.